app/src/main/java/org/mozilla/focus/navigation/TransactionHelper.kt

```diff
@@ -181,25 +183,24 @@ internal class TransactionHelper(private val activity: HostActivity) : DefaultLi
     }
 
     private fun getEntryTag(entry: FragmentManager.BackStackEntry): String {
-        val result = entry.name!!.split(ENTRY_TAG_SEPARATOR).toTypedArray()
-        return result[0]
+        return entry.name?.let {
+            it.split(ENTRY_TAG_SEPARATOR).toTypedArray()[0]
+        } ?: ""
```

Comment:
should we throw an error or return a default value when `entry.name` is null

Reply:
It will return "" when `entry.name` is null